### PR TITLE
feat: Avatar component created

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
Avatar component is created.
1. It is having small, medium, large and extra-large.
2. Simple and rounded avatars of the same above specification are created.
3. Please review the code and help in code improvement.

### CODE REVIEW

File: componentLibrary.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+/* *****avatar***** */
+.avatar-container {
+    max-width: 100%;
+    border: 1px solid black;
+    margin: 2rem;
+    padding: 2rem;
+    display: flex;
+    flex-direction: row;
```
Comment (mariyasada) [inline componentLibrary.css:23]: use flex property gap for adding space between two elements and use shorthand property flex-flow for flex-direction and flex-wrap like flex-flow: row wrap

File: component_documentation/componentDoc.css [hunk 2]
```diff
@@ -163,6 +167,38 @@ body {
...
+    margin-bottom: 1rem;
+}
+
+.sub-component-container {
+    box-sizing: border-box;
+    margin: 1rem;
+    padding: 1rem;
+    margin-bottom: 2rem;
+}
+
```
Comment (mariyasada) [inline component_documentation/componentDoc.css]: don't repeat border-box property.you can use body{box-sizing:border-box} in main css.

**Review by SaurabhChawda — CHANGES_REQUESTED** [review]
Reduce the size of the images.

**Review by mariyasada — APPROVED** [review]

File: componentLibrary.css (hunk 1 above)
Comment (ShreyashMahajan) [reply componentLibrary.css:23]: ok, i will do that

File: component_documentation/componentDoc.css (hunk 2 above)
Comment (ShreyashMahajan) [reply component_documentation/componentDoc.css]: thank you. Ok

**Comment by ShreyashMahajan** [comment]
> Reduce the size of the images.

ok

**Review by SaurabhChawda — COMMENTED** [review]
review approved

**Pull request merged**